reactor-netty-http/src/main/java/reactor/netty/http/client/Http2DelegatingConnectionPoolMetrics.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright (c) 2025 VMware, Inc. or its affiliates, All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package reactor.netty.http.client;
+
+
+final class Http2DelegatingConnectionPoolMetrics implements Http2ConnectionPoolMetrics {
```

Comment:
Let's call it only HttpDelegatingConnectionPoolMetrics as it will be able to be used with HTTP/2 or HTTP/3

```suggestion
final class HttpDelegatingConnectionPoolMetrics implements Http2ConnectionPoolMetrics {
```